InaraResponder/InaraResponder.cs

```diff
@@ -50,10 +41,18 @@ public bool Start()
             Reload();
 
             // Subscribe to events from the Inara configuration that require our attention
-            InaraConfiguration.ConfigurationUpdated += (s, e) => { OnConfigurationUpdated((InaraConfiguration)s); };
+            InaraService.invalidAPIkey += OnInvalidAPIkey;
 
             Logging.Info($"Initialized {ResponderName()}");
-            return inaraService != null;
+            return true;
+        }
+
+        private void OnInvalidAPIkey(object sender, EventArgs e) 
+        {
+            // Alert the user that there is a problem with the Inara API key
+            Logging.Info("API key is invalid: Please open the Inara Responder and update the API key.");
+            ShipMonitor shipMonitor = (ShipMonitor)EDDI.Instance.ObtainMonitor(EddiShipMonitor.Properties.ShipMonitor.ResourceManager.GetString("name", CultureInfo.InvariantCulture));
```

Comment:
I'm not a fan of audible announcements and reaching into the ShipMonitor like this is rather icky.

EDit: ah, it's moved from another method. I still don't like it but I'll let it slide for now.